### PR TITLE
Avatar command

### PR DESCRIPTION
**Pull request by Tienei**
Added avatar command

### CODE REVIEW

**Comment by Shompi** [comment]
Just a couple of suggestions
- Use BLUE as embed color
- No need to use async since there is no await statement.

Looks pretty good, im going to wait for the changes :smile:

**Comment by Tienei** [comment]
Update the changes, waiting for review owo

File: Commands/Commands/UserAvatar.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    usage: "avatar (@Mención de usuario | ID | nombre de usuario)",
+    nsfw: false,
+    enabled: true,
+    aliases: [],
+    permissions: [],
+
+    async execute(message = new Message(), args = new Array()) {
+        try {
+            const { guild, channel, mentions } = message;
+            const user = args.shift();
```
Comment (Shompi) [inline Commands/Commands/UserAvatar.js:50]: Would be better to use `args.join(" ");`
Reason: Usernames with more than 1 word.

File: Commands/Commands/UserAvatar.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                embed = getAuthorAvatar(message)
+            } else {
+                if (isNaN(user)) {
+                    // A String input
+                    if (mentions.users.array().length > 0) {
+                        // Mentioned
+                        embed = findUserByMention(message)
+                    } else {
+                        // Find user by name
+                        if (guild !== null) {
```
Comment (Shompi) [inline Commands/Commands/UserAvatar.js:62]:
```suggestion
                        if (guild) {
```

File: Commands/Commands/UserAvatar.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            return avatarEmbed(membersList[i].user)
+        }
+    }
+}
+
+module.exports = {
+    name: "avatar",
+    guildOnly: false,
+    filename: path.basename(__filename),
+    description: "Ver el avatar de un usuario",
```
Comment (Shompi) [inline Commands/Commands/UserAvatar.js:40]:
```suggestion
    description: "Envía el avatar de un usuario.",
```

**Comment by Shompi** [comment]
One last point:
Remove the `try catch` block since Im catching the errors in the main file.

**Comment by Tienei** [comment]
minor changes done

**Pull request merged**